github-actions/google-internal-tests/lib/main.ts

```diff
@@ -80,6 +100,22 @@ function constructPatterns(rawPatterns: string): minimatch.IMinimatch[] {
   return patterns;
 }
 
+async function findExistingTestStatus(
+  github: Octokit,
+  prHeadSHA: string,
+): Promise<GithubStatus | null> {
+  const existingStatuses = await github.paginate(
+    github.repos.getCombinedStatusForRef,
```

Comment:
Can this method even paginate?

Reply:
Yes, looks like it: https://octokit.github.io/rest.js/v19#:~:text=Get%20the%20combined%20status%20for%20a%20specific%20reference